api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -285,4 +275,17 @@ public boolean supportsPropertyName(String gpName) {
 	public List<String> findSimilarFamilyNames(String searchPhrase) {
 		return getPatientNameSearch().findSimilarFamilyNames(searchPhrase);
 	}
-}
+
+	@Override
+	public String importMpiPatient(String personId) {
+		if (coreProperties.isMpiEnabled()) {
+			MpiProvider mpiProvider = coreProperties.getMpiProvider();
+			Patient importedPatient = mpiProvider.importMpiPatient(personId);
```

Comment:
looking at this method call... I would have expected a method called "importMpiPatient" to fetch the patient from the MPI _and_ save them to the database. If we're not actually saving them, I would just call it "fetchMpiPatient"